re-enable, upgrade selenium-server-standalone-2.30.0.jar

isao opened the pull request:
will this go green?

Commit:
re-enable, upgrade selenium-server-standalone-2.30.0.jar

check if selenium is ok on travis

drewfish commented:
+1

BTW, upgrading selenium is a prerequisite to upgrading yahoo-arrow beyond 0.0.73.

Commit:
comment out fund tests

need to change func test execution so it waits (longer)
until mojito client is ready
